src/main/resources/templates/macros/report/docstring.vm

```diff
@@ -8,7 +8,7 @@
           <a>Doc string</a>
       </div>
       <div id="msg-$docStringId" class="collapse collapsable-details">
-        <pre>$docString.getValue()</pre>
+        <pre>$escHtml.translate($docString.getValue())</pre>
```

Comment:
Not really. Developer can use this option to display any HTML content so they can control how the browser renders it (eg display results in HTML table). If they want to see HTML code as escaped content they format that content before storing into the report.

By this change you don't allow developers to display additional information as HTML table, formatted text or image loaded from external server